tool/tests/mock/mock_ecu.py

```diff
@@ -10,10 +11,7 @@ class MockEcu:
 
     def __init__(self, bus=None):
         self.message_process = None
-        if bus is None:
```

Comment:
Just like in `tool/lib/iso15765_2.py` - is this logic handled elsewhere now, to prevent `CanError` when called with a virtual CAN bus while the OS is lacking a working CAN interface?